src/lib/settings/settings.py

```diff
@@ -287,10 +414,11 @@ def from_file_path(cls, base_path: str):
             monitoring_settings,
             recipients_settings,
             replacements_settings,
+            iam_role_extract_metrics,
         )
 
     @classmethod
-    def from_s3_path(cls, base_path: str):
+    def from_s3_path(cls, base_path: str, iam_role_extract_metrics: str = None):
```

Comment:
Maybe we can have another name here, because inside the Settings class we don't extract metrics :)
rather it's a role to access and list monitored resources in general.

From my perspective, it might become misleading

Reply:
Done